sway/desktop/transaction.c

```diff
@@ -420,13 +420,19 @@ static void transaction_commit(struct sway_transaction *transaction) {
 		struct sway_transaction_instruction *instruction =
 			transaction->instructions->items[i];
 		struct sway_node *node = instruction->node;
+		bool hidden = node_is_view(node) &&
+			container_is_hidden(node->sway_container);
```

Comment:
Hmm, how does this differ from `!view_is_visible(node->sway_container->view)`?

Reply:
Argh, didn't see that it was implemented on sway_view. It doesn't belong there, as all the tested properties are container properties. :/

The view version is a bit more expensive and tests for scenarios that likely don't matter here, but it's silly to have two, so I'll switch over.

Reply:
Agreed it's a bit confusing. I think the logic goes like this, though: if you have `T[A* B]`, do you expect `container_is_visible(B)` to return true? In some sense it should, because the titlebar is visible. But, its contents are not, so it's not useful in the common case.

Perhaps this is just a naming thing though, and calling it `container_view_is_visible` and moving it to `container.h` would be fine.